### PR TITLE
VTCCA-2835 resolving issues with agent representation (still requires…

### PR DESCRIPTION
**Pull request by DanielConnelly**
… revoke and non agent query)

### CODE REVIEW

File: app/connectors/propertyLinking/PropertyLinkConnector.scala [hunk 1]
```diff
@@ -72,20 +72,30 @@ class PropertyLinkConnector @Inject()(config: ServicesConfig, http: WSHttp)(impl
...
-          "params.pageSize" -> pagination.startPoint.toString,
-          "params.requestTotalRowCount" -> pagination.requestTotalRowCount.toString))
+    val ownerAuthResult = http.GET[OwnerAuthResult](
+      s"$baseUrl/$ownerOrAgent/property-links",
+      List(
+        searchParams.address.map("address" -> _),
+        searchParams.baref.map("baref" -> _),
+        searchParams.agent.map("agent" -> _),
+        searchParams.status.map("status" -> _),
+        searchParams.status.map("status" -> _),
```
Comment (habeeb01) [inline app/connectors/propertyLinking/PropertyLinkConnector.scala]: status is duplicated

File: app/views/propertyrepresentation/appointAgentProperties.scala.html [hunk 2]
```diff
@@ -74,15 +73,15 @@ <h1 class="heading-large">@Messages("propertyRepresentation.appointAgentProperti
...
+                <input type="hidden" name="checkPermission" value="@checkPermission">
+                <input type="hidden" name="challengePermission" value="@challengePermission">
+                <input type="hidden" name="pageNumber" value="@pagination.page">
                 <input type="hidden" name="pageSize" value="@pagination.pageSize">
-                <input type="hidden" name="sortField" value="@pagination.sortField">
-                <input type="hidden" name="sortOrder" value="@pagination.sortOrder">
-                <input type="hidden" name="agentAppointed" value="@pagination.agentAppointed">
+                <input type="hidden" name="sortField" value="@params.sortfield">
+                <input type="hidden" name="sortOrder" value="@params.sortorder">
+                <input type="hidden" name="agentAppointed" value="BOTH">
```
Comment (habeeb01) [inline app/views/propertyrepresentation/appointAgentProperties.scala.html]: why is this hard-coded

File: project/FrontendBuild.scala [hunk 3]
```diff
@@ -15,12 +15,12 @@ object FrontendBuild extends Build with MicroService {
...
 
   override lazy val playSettings: Seq[Setting[_]] = Seq(
-    routesImport ++= Seq("models.SortOrder", "models.messages.MessagePagination", "models.searchApi.AgentPropertiesParameters"),
+    routesImport ++= Seq("binders._", "binders.pagination._", "models.SortOrder", "models.messages.MessagePagination", "models.searchApi.AgentPropertiesParameters"),
     // Add the views to the dist
     unmanagedResourceDirectories in Assets += baseDirectory.value / "app" / "assets",
     // Dont include the source assets in the dist package (public folder)
     excludeFilter in Assets := "fonts" || "tasks" || "karma.conf.js" || "tests" || "gulpfile.js*" || "js*" || "src*" || "node_modules*" || "sass*" || "typescript*" || "typings*" || ".jshintrc" || "package.json" || "tsconfig.json" || "tsd.json"
-  )  ++ JavaScriptBuild.javaScriptUiSettings
+  )  //++ JavaScriptBuild.javaScriptUiSettings
```
Comment (habeeb01) [inline project/FrontendBuild.scala]: javascript

**Comment by DanielConnelly** [comment]
@jon-pike what is the question on the logging?

**Pull request merged**